packages/aws-cdk-lib/aws-ec2/lib/ip-addresses.ts

```diff
@@ -144,9 +152,18 @@ export interface SubnetIpamOptions {
  */
 export interface AllocatedSubnet {
   /**
-   * Cidr Allocations for a Subnet
+   * IPv4 Cidr Allocations for a Subnet.
+   *
+   * Note this is specific to the IPv4 CIDR.
    */
   readonly cidr: string;
+
+  /**
+   * IPv6 Cidr Allocations for a Subnet.
+   *
+   * Note this is specific to the IPv6 CIDR.
+   */
+  ipv6Cidr?: string;
```

Comment:
Should this be readonly, like the IPv4 version? (P.S. Nice to see the documentation updates to the IPv4 only fields).

Also, should this be a collection (array/list), because multiple IPv6 subnets (like multiple IPv6 addresses) will be much more common, i.e. make the default with support for multiple IPv6. (Yes, you can have multiple IPv4, but that would be much less common).